gensim/models/keyedvectors.py

```diff
@@ -742,10 +742,14 @@ def most_similar(
             clip_start = 0
             clip_end = restrict_vocab
 
-        if isinstance(positive, KEY_TYPES) and not negative:
-            # allow calls like most_similar('dog'), as a shorthand for most_similar(['dog'])
+        if isinstance(positive, KEY_TYPES + (ndarray,)):
+            # allow passing a single string-key or vector for the positive argument
```

Comment:
Is there a risk that something that may have previously worked, like an ndarray of multiple vectors, will stop working here, with the extra `list` wrapping?

Reply:
Still a concern here.

Reply:
Still a concern here.

Reply:
@gojomo @piskvorky How do you suggest we resolve these concerns?

Reply:
@gojomo @piskvorky Ping. This is blocking the release.

Reply:
Check if ndarray-of-multiple-vectors used to work as any of these arguments, and if so, if it's now broken. If so, then decide whether to fix to retain old behavior for that usage, or just add to release-note or doc-comment as rare case no longer supported.

Reply:
I checked and `model.most_similar(positive=np.array([vec1, vec2]))` indeed does work in `develop`. And will likely stop working here.

Reply:
OK. Finally had a chance to look at this. Please have a look at let me know if I've missed anything.